flow/artifactstorage/artifactory/artifactory.py

```diff
@@ -379,3 +422,10 @@ def _download_and_extract_artifact_locally(self, download_dir, extension, extrac
         print(download_path)
 
         commons.print_msg(Artifactory.clazz, method, 'end')
+
+    def check_artifact_permissions(self, remove_resp, method):
```

Comment:
Can we start the method name with "_"?  I've been sticking with that pattern for private methods.

Reply:
Should be better now 👍